conjur/cli.py

```diff
@@ -81,7 +81,7 @@ def command_epilog(example, command=None, subcommands=None):
         """
         This method builds the footer for each command help screen.
         """
-        refer_to_help = "See more details in each subcommand’s help:"
+        refer_to_help = "See more details in each subcommand\'s help:"
```

Comment:
the backslash is not a mistake?

Reply:
I added it in because I thought it would error but I see now it doesnt and will only error when I use `''` over `""`